Plugin does not always check for PAPI before calling it

When Message.broadcast() is called the presence of Placeholder API should always be checked before using it to replace the placeholders.
For example, this check is not done [here](https://github.com/Oheers/EvenMoreFish/blob/3537d9f5e5e6864d59be67f6c6d5e5f4f1e8273b/even-more-fish-plugin/src/main/java/com/oheers/fish/config/messages/Message.java#L178) leading to unhandled exceptions when executing commands as `/evenmorefish admin nbt-rod` without the other plugin being loaded.

Example of the error:
`org.bukkit.command.CommandException: Unhandled exception executing command 'evenmorefish' in plugin EvenMoreFish v1.6.11.17
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[spigot-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[spigot-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_20_R3.CraftServer.dispatchCommand(CraftServer.java:875) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at org.bukkit.craftbukkit.v1_20_R3.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:50) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at com.mojang.brigadier.context.ContextChain.runExecutable(ContextChain.java:73) ~[brigadier-1.2.9.jar:?]
	at net.minecraft.commands.execution.tasks.ExecuteCommand.a(SourceFile:29) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.commands.execution.tasks.ExecuteCommand.execute(SourceFile:13) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.commands.execution.UnboundEntryAction.a(SourceFile:8) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.commands.execution.CommandQueueEntry.a(SourceFile:8) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.commands.execution.ExecutionContext.a(SourceFile:107) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.commands.CommandDispatcher.a(CommandDispatcher.java:413) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.commands.CommandDispatcher.performCommand(CommandDispatcher.java:335) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.commands.CommandDispatcher.a(CommandDispatcher.java:322) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1856) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.server.network.PlayerConnection.lambda$15(PlayerConnection.java:1818) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.util.thread.IAsyncTaskHandler.b(SourceFile:67) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at net.minecraft.server.TickTask.run(SourceFile:18) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.util.thread.IAsyncTaskHandler.d(SourceFile:156) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(SourceFile:23) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1191) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:1) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.util.thread.IAsyncTaskHandler.x(SourceFile:130) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.server.MinecraftServer.bl(MinecraftServer.java:1170) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1163) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.util.thread.IAsyncTaskHandler.c(SourceFile:139) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.server.MinecraftServer.w_(MinecraftServer.java:1147) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1060) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:304) ~[spigot-1.20.4-R0.1-SNAPSHOT.jar:4029-Spigot-c198da2-3337018]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.NoClassDefFoundError: me/clip/placeholderapi/PlaceholderAPI
	at com.oheers.fish.config.messages.Message.broadcast(Message.java:178) ~[?:?]
	at com.oheers.fish.Controls.adminControl(CommandCentre.java:493) ~[?:?]
	at com.oheers.fish.CommandCentre.control(CommandCentre.java:186) ~[?:?]
	at com.oheers.fish.CommandCentre.onCommand(CommandCentre.java:84) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	... 29 more
Caused by: java.lang.ClassNotFoundException: me.clip.placeholderapi.PlaceholderAPI
	at com.oheers.fish.config.messages.Message.broadcast(Message.java:178) ~[?:?]
	at com.oheers.fish.Controls.adminControl(CommandCentre.java:493) ~[?:?]
	at com.oheers.fish.CommandCentre.control(CommandCentre.java:186) ~[?:?]
	at com.oheers.fish.CommandCentre.onCommand(CommandCentre.java:84) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-api-1.20.4-R0.1-SNAPSHOT.jar:?]
	... 29 more`

--- a/even-more-fish-plugin/src/main/java/com/oheers/fish/EvenMoreFish.java
+++ b/even-more-fish-plugin/src/main/java/com/oheers/fish/EvenMoreFish.java
@@ -215,7 +215,6 @@ public class EvenMoreFish extends JavaPlugin implements EMFPlugin {
         AutoRunner.init();
 
         wgPlugin = getWorldGuard();
-        checkPapi();
 
         if (mainConfig.databaseEnabled()) {
 

--- a/even-more-fish-plugin/src/main/java/com/oheers/fish/EvenMoreFish.java
+++ b/even-more-fish-plugin/src/main/java/com/oheers/fish/EvenMoreFish.java
@@ -103,7 +103,6 @@ public class EvenMoreFish extends JavaPlugin implements EMFPlugin {
 
     public static WorldGuardPlugin wgPlugin;
     public static String guardPL;
-    public static boolean papi;
     public static DatabaseV3 databaseV3;
     public static HeadDatabaseAPI HDBapi;
     private static EvenMoreFish instance;
@@ -158,7 +157,7 @@ public class EvenMoreFish extends JavaPlugin implements EMFPlugin {
             guiFillerStyle = guiConfig.getFillerStyle("main-menu");
         }
 
-        usingPAPI = getServer().getPluginManager().getPlugin("PlaceholderAPI") != null;
+        checkPapi();
 
         if (mainConfig.requireNBTRod()) {
             customNBTRod = createCustomNBTRod();
@@ -535,7 +534,7 @@ public class EvenMoreFish extends JavaPlugin implements EMFPlugin {
 
     private void checkPapi() {
         if (Bukkit.getPluginManager().getPlugin("PlaceholderAPI") != null) {
-            papi = true;
+            usingPAPI = true;
             new PlaceholderReceiver(this).register();
         }
 

--- a/even-more-fish-plugin/src/main/java/com/oheers/fish/config/messages/Message.java
+++ b/even-more-fish-plugin/src/main/java/com/oheers/fish/config/messages/Message.java
@@ -138,7 +138,7 @@ public class Message {
         if (doColour) colourFormat();
         if (this.message.endsWith(" -s") && this.canSilent) return;
 
-        if (relevantPlayer != null) this.message = PlaceholderAPI.setPlaceholders(relevantPlayer, this.message);
+        if (relevantPlayer != null && EvenMoreFish.usingPAPI) this.message = PlaceholderAPI.setPlaceholders(relevantPlayer, this.message);
 
         for (Player p : Bukkit.getOnlinePlayers()) {
             p.sendMessage(this.message);
@@ -157,7 +157,7 @@ public class Message {
         if (doColour) colourFormat();
         if (this.message.endsWith(" -s") && this.canSilent) return;
 
-        if (relevantPlayer != null && EvenMoreFish.papi) this.message = PlaceholderAPI.setPlaceholders(relevantPlayer, this.message);
+        if (relevantPlayer != null && EvenMoreFish.usingPAPI) this.message = PlaceholderAPI.setPlaceholders(relevantPlayer, this.message);
 
         player.sendMessage(this.message);
     }
@@ -175,7 +175,7 @@ public class Message {
         if (doColour) colourFormat();
         if (this.message.endsWith(" -s") && this.canSilent) return;
 
-        if (relevantPlayer != null) this.message = PlaceholderAPI.setPlaceholders(relevantPlayer, this.message);
+        if (relevantPlayer != null && EvenMoreFish.usingPAPI) this.message = PlaceholderAPI.setPlaceholders(relevantPlayer, this.message);
 
         sender.sendMessage(this.message);
     }

--- a/even-more-fish-plugin/src/main/java/com/oheers/fish/config/messages/OldMessage.java
+++ b/even-more-fish-plugin/src/main/java/com/oheers/fish/config/messages/OldMessage.java
@@ -106,7 +106,7 @@ public class OldMessage {
 
     public String toString() {
 
-        if (EvenMoreFish.papi) {
+        if (EvenMoreFish.usingPAPI) {
             if (receiver != null) {
                 msg = PlaceholderAPI.setPlaceholders(receiver, msg);
             }

